camel/__init__.py

```diff
@@ -6,7 +6,7 @@
 import camel.typing
 import camel.utils
 
-__version__ = '0.0.1'
+__version__ = '0.1.0'
```

Comment:
Revert. This is irrelevant to the ticket. Please make another ticket to fix this bug.